src/main/java/FeatureTest.java

```diff
@@ -4,6 +4,9 @@
 public class FeatureTest {
     public static void main(String[] args) {
         log.info("Hello, World!");
+        log.info("More");
+        String string = null;
+        log.info(string.lenght());
```

Comment:
Null pointer dereference: Variable 'string' is null (assigned null at line 8)

Reply:
resolved